### PR TITLE
add compression keyword

### PR DESCRIPTION
**Pull request by Genius-pig**
add compression keyword

### CODE REVIEW

File: docs/UserGuide/Operation Manual/SQL Reference.md [hunk 1]
```diff
@@ -32,14 +32,12 @@ All of these statements are write in IoTDB's own syntax, for details about the s
 
 ## Keywords
 
-Please not use these keywords as identifiers.
+Please not use these keywords as identifiers. If you really want to use these keywords in your iotdb path, please contact us.
```
Comment (jixuan1989) [inline docs/UserGuide/Operation Manual/SQL Reference.md:35]: this description is not suitable.
```
Do not use the following keywords as identifiers. 
You can open an [issue](https://issues.apache.org/jira/projects/IOTDB/issues) to provide new requirements or suggestions. 
```

File: docs/zh/UserGuide/Operation Manual/SQL Reference.md [hunk 2]
```diff
@@ -23,14 +23,12 @@
 
 ## 关键字
 
-不要使用这些关键字作为标识符。
+不要使用这些关键字作为标识符。如果你真的想需要使用这些关键字在你的iotdb路径上，请联系我们。
```
Comment (jixuan1989) [inline docs/zh/UserGuide/Operation Manual/SQL Reference.md:26]: 不要使用这些关键字作为标识符。
如果有需求或者建议，可以在[issue](https://issues.apache.org/jira/projects/IOTDB/issues) 上提出。

**Comment by jixuan1989** [comment]
pls solve the conflict.

I concern is, we do not allow users to use keywords, but there are too many keywords...

File: docs/UserGuide/Operation Manual/SQL Reference.md (hunk 1 above)
Comment (Genius-pig) [reply docs/UserGuide/Operation Manual/SQL Reference.md]: done.

**Pull request merged**